frontend/src/components/auth/components/sign-in-form/styles.module.scss

```diff
@@ -33,7 +33,7 @@
 }
 
 .buttonWrapper {
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  width: 93px;
+  height: 52px;
```

Comment:
can we define these as min-width and min-height in btn component? Don't want to define them each type when using btn

Reply:
We can, but if in future there'll be smaller buttons, everything will have to be changed back, but I'll change it after I create a helper for student pages :)

Reply:
what about min-width with alignSelf?

Reply:
And why do we need to change button height?

Reply:
@Liza-Veis I have tried with align-self, button still takes the whole width. And I've given to the wrapper `display: flex`, because align-self only works inside of flex
About the height, we need it, because otherwise the buttons will look like this:
![image](https://user-images.githubusercontent.com/61088675/189203260-64d9f67e-63de-4bc6-b315-5e0b61d2b4ee.png)

I'll use `min-height` for button and remove from all styles, but I think width should stay as it is

Reply:
Add padding to button component to avoid issue with height

Reply:
I meant you should add align-self or margin-left auto for wrapper, and then button should fit it's width

Reply:
@Liza-Veis it can be done by using min-width for button and align-self for wrapper, but in my opinion, it complicates the situation, because using align-self means, that we have to add styles to the wrapper of button wrapper to ensure, that it has `display: flex;`. May I know, why do you prefer this approach over the one which uses width?

Reply:
Just saw, that adding min-width will break all the `IconButton`, because they use `Button` component. The Icons will use a lot of space, which will break the view.